web/server/contract/result.go

```diff
@@ -104,16 +101,6 @@ func NewTestResult(testName string) *TestResult {
 	return self
 }
 
-func resolvePackageName(path string) string {
```

Comment:
If you look at the callsite for this function, you can see this is is run on *messaging.Folder.Path. Would it be possible to add a new field to *messaging.Folder which explicitly has Name? That way the 1.9 split only needs to be in the convey package.